CHANGELOG.md

```diff
@@ -1,13 +1,160 @@
 # Smithy Changelog
 
-## 0.10.0 (2020-??-??)
+## 1.0.0 (2020-??-??)
+
+***Note***: Changes marked with "[BC]" are breaking changes more accurately described in the
+specific section. A list of further intended breaking changes have a specific section near
+the end of this entry.
+
+### Features
+
+#### General
+
+* The model format version has beeen updated to `1.0.0` and contains several updates: [BC] ([#357](https://github.com/awslabs/smithy/pull/357), [#381](https://github.com/awslabs/smithy/pull/381))
+  * The JSON AST representation requires describing annotation traits as `{}` instead of `true`.
+  * The IDL representation requires describing annotation traits without `true` or `null` values.
```

Comment:
Annotation traits in the IDL are now provided as `@foo` or `@foo()`. Explicit true or null support was removed.